modules/b2b_logic/logic.c

```diff
@@ -3304,9 +3304,6 @@ str* b2b_process_scenario_init(struct sip_msg* msg, b2bl_cback_f cbf,
 	pkg_free(server_id);
 	tuple->servers[0]->type = B2B_SERVER;
 
-	pkg_free(to_uri.s);
-	to_uri.s = 0;
```

Comment:
Removing this would introduce a memory leak. Can you move this code after `b2b_api.client_new()` ?

Reply:
@rvlad-patrascu thank you for reviewing. I fixed it as you recommended.